src/test/run-pass/irrefutable-slice-patterns.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// #47096
+
+#![feature(slice_patterns)]
+#![allow(dead_code)]
+
+fn foo(s: &[i32]) {
+    let &[ref _xs..] = s;
```

Comment:
Can we update this test to actually use `xs`?

For example:

```rust
fn foo(s: &[i32]) -> &[i32] {
    let &[ref s1..] = s;
    s1
}

fn main() {
    let x = [1, 2, 3];
    let y = foo(&x);
    assert_eq!(y, &[1, 2, 3]);
}
```

Reply:
Oops, I forgot to do this.